### ISSUE TITLE
Drag file from FileExplorer to WebView(inspector)

### ISSUE THREAD

**Issue by leanmendoza** [decentraland/sdk#753]
Figure out if is something possible to do or if the feature is blocked by the WebView API for extension in Visual Studio Code.

Example: Drag a file from the `Finder` directly to `Import Asset` tab (it works if the Inspector is opened from the Browser)

**Comment by nearnshaw** [comment]
@leanmendoza  I think this is already working, right?
or am I understanding it wrong?

**Comment by leanmendoza** [comment]
> @leanmendoza I think this is already working, right? or am I understanding it wrong?

I don't think so, you need to put the asset in your scene project, and then drag it from the Asset import tab, right?

**Comment by nearnshaw** [comment]
This is already done
We can import a 3D model into the local assets folder
What's missing is a whole redesign of the import process, but that's TBD via Shape up
